CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.100.0](https://github.com/awslabs/aws-solutions-constructs/compare/v1.99.0...v1.100.0) (2021-04-28)
+
+### Changed
+- Upgraded all patterns to CDK v1.100.0
+- Fixed issue with refreshing Integration Tests leaving Resources Behind in Account ([#164](https://github.com/awslabs/aws-solutions-constructs/pull/164))
```

Comment:
Probably should document that this is a potentially breaking change - if a stack is currently sending both an existingBucket and bucketProps to a construct, previous behavior was to ignore the props silently. Now the launch will fail.